lstchain/scripts/lstchain_r0_to_r0g.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python
```

Comment:
I'd add here a comment explaining which version of data this is expected to be used for (EvB6, as of date, etc)